src/write_http.c

```diff
@@ -80,6 +80,21 @@ struct wh_callback_s
 };
 typedef struct wh_callback_s wh_callback_t;
 
+static void wh_log_http_error_empty(wh_callback_t *cb){}
+
+void (*wh_log_http_error)(wh_callback_t *cb) = wh_log_http_error_empty;
```

Comment:
This function pointer is too hard to read and doesn't provide enough benefit to justify it. Please just have a (module-)global config variable and put the *if (status != 200 && log_http_errors)* logic into `wh_log_http_error_impl`.